spec/models/enrollment_spec.rb

```diff
@@ -25,6 +25,18 @@
     end
   end
 
+  describe 'validations' do
+    context 'when a new enrollment is created where mission is full' do
+      it 'raises an error' do
+        mission =  create :mission, max_member_count: 4 
+        enroll_n_members_on_mission(mission, 4)
+        extra_enrollment = Enrollment.new(mission: mission)
+
+        expect(extra_enrollment).to_not be_valid
+      end
```

Comment:
Instead of creating a method helper that is used only once in that file (meaning, it only moves the logic elsewhere), I think this setup can be done directly under this context. You could either create a new `trait` in the `mission` factory, or directly set the setup here in a `let`. Here, the `let` directly illustrates what is the actual context in terms of code

```suggestion
      let(:full_mission) do
        create :mission, max_member_count: 4 do |mission|
            create_list :enrollment,
                         4,
                         mission: mission,
                         start_time: mission.start_time,
                         end_time: mission.start_date + 3.hours
          end
        end
      end
      
      it 'is invalid' do
        extra_enrollment = Enrollment.new(mission: full_mission)

        expect(extra_enrollment).to_not be_valid
      end
```

Also, it's better to be specific if we can on what error is raised. Here, it's an invalidation